index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Survey Form</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+  <h1 id="title">Survey Form</h1>
+  <p id="description">
+    Thank you for taking the time to help us improve the platform
+  </p>
+  <div class="form-container">
```

Comment:
It is best practice to use semantic tags in html. I suggest changing this div tag to section semantic tag.

Reply:
- [ ] Also you may consider providing some resources on the internet that can help the author to get the concept a 100% because things may be clear to you but not to the other person. Kindly you may consider that.